src/App.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Categories from './components/Categories';
+import Books from './components/Books.';
```

Comment:
- [OPTIONAL] Everything is perfect 💯, So I marked this small issue as optional, Just there is a small typo in this line `import Books from './components/Books.';` Kindly, remove the dot at the end of the line 🙏😅